crates/wash-cli/tests/wash_call.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::{Context, Result};
+use serial_test::serial;
+
+use wash_lib::cli::output::StartCommandOutput;
+
+mod common;
+use common::{TestWashInstance, HTTP_JSONIFY_OCI_REF};
+
+/// Ensure that wash call works
+#[tokio::test]
+#[serial]
+#[cfg_attr(
+    not(can_reach_wasmcloud_azurecr_io),
+    ignore = "wasmcloud.azurecr.io is not reachable"
+)]
+async fn integration_call() -> Result<()> {
+    let instance = TestWashInstance::create().await?;
+
+    // Start an echo component
+    let StartCommandOutput { component_id, .. } = instance
+        .start_component(HTTP_JSONIFY_OCI_REF, "http-jsonify")
+        .await
+        .context("failed to start component")?;
+    let component_id = component_id.context("component ID not present after starting component")?;
+
+    // Build request payload to send to the echo component
+    let request = serde_json::json!({
+        "method": "GET",
+        "path": "/",
+        "body": "",
+        "queryString": "",
+        "header": {},
+    });
+
+    // TODO: add test when no input data is specified as well
+    // (should probably turn into a general GET rquest")
+
```

Comment:
This is the default right? Or is this just a note for the jsonify component

Reply:
Yup this is the default if nothing is specified, removed the TODO (and greatly expanded customization)